source/_components/rainbird.markdown

```diff
@@ -40,8 +42,67 @@ password:
   description: The password for accessing the module.
   required: true
   type: string
+trigger_time:
+  description: Irrigation time.
+  required: true
```

Comment:
You've changed this is your last commit and doesn't match the code anymore.
Either, revert this or adjust the code in the Home Assistant main code base PR.

Reply:
Now it matches